recipe/meta.yaml

```diff
@@ -32,24 +32,24 @@ requirements:
     - packaging
     - pip
     - python
-    - setuptools
```

Comment:
`setuptools` is still implicitly in `host` so no need to remove it. See also https://github.com/jupyterlab/jupyterlab/blob/3dea0ef213db2b44b5634efc20245e0034744932/setup.py#L14